src/screens/Settings.js

```diff
@@ -0,0 +1,195 @@
+
+const Settings = ({modalVisible, hideModal}) => {
+    const [toEmailType, setToEmailType] = useState("to");
+    const [toEmail, setToEmail] = useState("");
+    const [emails, setEmails] = useState([
+        // sample default email
+        // TODO: add integration for fetching  settings/emails
+        {
+            email: 'sample@gmail.cm',
+            type: 'to'
+        },
+        {
+            email: 'sample2@gmail.cm',
+            type: 'cc'
+        }
+
+    ]);
+    const { serviceSelected } = useContext(ServiceContext);
+
+    const mode = serviceSelected;
+
+    const addEmail = email => setEmails([...emails, email]);
+
+    const removeEmail = indexToRemove => {
+        let tempEmails = emails.filter((value, index) => {
+            return index !== indexToRemove;
+        });
+
+        setEmails(tempEmails);
+    }
+
+    const addEmailToList = () => {
+        // todo: validate email
+
+        if (toEmail.length) {
+            const email = {
+                email: toEmail,
+                type: toEmailType
+            };
+
+            addEmail(email);
+        }
+
+        setToEmail("");
+    }
+
+    // TODO add submit button for API integration
+    return (
+        <Modal
+            animationType="slide"
+            visible={modalVisible}
+            onRequestClose={() => {
+                hideModal(false);
+            }}
+        >
+            <View style={[styles.container, globalStyles.containerBg[mode]]}>
+                <View style={styles.headerContainer}>
+                    <Text style={styles.headerText}>Settings</Text>
+                </View>
+                <View style={styles.formContainer}>
+                    <View style={styles.inputContainer}>
+                        <Text style={globalStyles.containerBg[mode]}>Pomodoro</Text>
+                        <TextInput style={[styles.timerInput, globalStyles.containerBg[mode]]} placeholder="pomodoro timer" placeholderTextColor="white" />
+                    </View>
+                    <View style={styles.inputContainer}>
+                        <Text style={globalStyles.containerBg[mode]}>Short break</Text>
+                        <TextInput style={[styles.timerInput, globalStyles.containerBg[mode]]} placeholder="short timer" placeholderTextColor="white" />
+                    </View>
+                    <View style={styles.inputContainer}>
+                        <Text style={globalStyles.containerBg[mode]}>Long break</Text>
+                        <TextInput style={[styles.timerInput, globalStyles.containerBg[mode]]} placeholder="long timer" placeholderTextColor="white" />
+                    </View>
+                </View>
+                <View style={styles.formContainer}>
+                    <View>
+                        <Text style={globalStyles.containerBg[mode]}>Emails to send to:</Text>
+                    </View>
+                    <View style={styles.emailList}>
+                        <FlatList
```

Comment:
insert keyExtractor prop. This will return a warning that a key was not generated if left undone.